src/compiler/config/outputs/validate-dist.ts

```diff
@@ -137,7 +137,9 @@ const validateOutputTargetDist = (config: d.ValidatedConfig, o: d.OutputTargetDi
     copy: validateCopy(o.copy ?? [], []),
     polyfills: isBoolean(o.polyfills) ? o.polyfills : undefined,
     empty: isBoolean(o.empty) ? o.empty : true,
-    transformAliasedImportPathsInCollection: o.transformAliasedImportPathsInCollection ?? false,
+    transformAliasedImportPathsInCollection: isBoolean(o.transformAliasedImportPathsInCollection)
+      ? o.transformAliasedImportPathsInCollection
+      : false,
```

Comment:
Should default `true` now, no?
```suggestion
    transformAliasedImportPathsInCollection: isBoolean(o.transformAliasedImportPathsInCollection)
      ? o.transformAliasedImportPathsInCollection
      : true,
```

Reply:
Yuppppp. Unsure how that slipped by, good catch!